lib/api/unittest/CDataFrameAnalyzerFeatureImportanceTest.cc

```diff
@@ -289,9 +384,41 @@ struct SFixture {
 
 template<typename RESULTS>
 double readShapValue(const RESULTS& results, std::string shapField) {
-    shapField = maths::CTreeShapFeatureImportance::SHAP_PREFIX + shapField;
-    if (results["row_results"]["results"]["ml"].HasMember(shapField)) {
-        return results["row_results"]["results"]["ml"][shapField].GetDouble();
+    if (results["row_results"]["results"]["ml"].HasMember("feature_importance")) {
+        for (const auto& shapResult :
+             results["row_results"]["results"]["ml"]["feature_importance"].GetArray()) {
```

Comment:
I generally prefer avoiding hidden dependency on field name in the unit tests if we can. I don't think this test needs to fail if we change the name. If we expose the string in `CDataFrameTrainBoostedTreeRunner` then let's use that definition.